packages/calypso-build/README.md

```diff
@@ -20,8 +20,8 @@ Then, add a `build` script that invokes the `calypso-build` command:
 
 ```json
 	"devDependencies": {
-        "@automattic/calypso-build": "1.0.0-beta.5"
-    },
+		"@automattic/calypso-build": "1.0.0-beta.5"
+	},
```

Comment:
It's presence in `devDependencies` is covered above by `npm install`… let's remove it here as this will always be out of date.

Reply:
9103eee